doc/classes/Joint3D.xml

```diff
@@ -4,7 +4,7 @@
 		Base class for all 3D joints.
 	</brief_description>
 	<description>
-		Joints are used to bind together two physics bodies. They have a solver priority and can define if the bodies of the two attached nodes should be able to collide with each other.
+		Joints are used to bind together two physics bodies. They have a solver priority and can define if the bodies of the two attached nodes should be able to collide with each other. See also [Generic6DOFJoint3D].
 	</description>
```

Comment:
I think you got a bit cut and past happy here.

Reply:
I know it wasn't really relevant here, but I added it just in case.